spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryCommentsConverter.java

```diff
@@ -531,6 +531,7 @@ private void convertReactionInformation(
                 reactionReferences.stream()
                         .filter(rr -> ReactionDatabase.RHEA.equals(rr.getDatabase()))
                         .map(CrossReference::getId)
+                        .filter(id -> !id.startsWith("RHEA-COMP"))
                         .collect(Collectors.toList());
```

Comment:
This is fine but i would have done differently to avoid dependency on the upper case.
I would have converted the id in lower/upper case and the checked starts with upper/lower case.